gobblin-core/src/main/java/gobblin/source/extractor/extract/restapi/RestApiExtractor.java

```diff
@@ -73,6 +73,27 @@ public RestApiExtractor(WorkUnitState state) {
 
   protected abstract RestApiConnector getConnector(WorkUnitState state);
 
+  protected void buildQuery(String inputQuery, String entity, JsonArray columnArray) {
```

Comment:
Fixed by renaming `buildQuery()` to `buildDataQuery()` and change return type to `String`
I don't know why `updatedQuery` was named in this way.
By tracing code, `updatedQuery` is used in `getDataMetadata()`, which will prepare query that is used to call GET command and get data from source.
Therefore I rename to data query in both method name and log.
However I can't rename `updatedQuery` to  `dataQuery` since it will affect subclass of `RestApiExtractor`